feature/networkinstance/networkinstance.go

```diff
@@ -49,9 +52,24 @@ func (ni *NetworkInstance) validate() error {
 	if ni.oc.GetType() == oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_UNSET {
 		return errors.New("NetworkInstance type is unset")
 	}
+	if ni.oc.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").GetOrCreateStatic(prefix).GetOrCreateNextHop("0").NextHop == oc.UnionString("") {
```

Comment:
ah I see why you made prefix global. This is not right as there should not be any global variables in this file. I believe what you want to validate here is that every prefix contains atleast one valid next-hop. The way to do it is by walking the config. Something like'

```
p := ni.oc.GetProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static")
for _, v := p.Static {
   for _, nh := v.NextHop {
      if nh == "" {
          // Fail
      }
   }
}
}
```

Reply:
Done!